cmd/builder/.goreleaser.yml

```diff
@@ -17,12 +17,11 @@ builds:
       - darwin
     goarch:
       - amd64
-      - arm
       - arm64
-    goarm:
-      - 6
-      - 7
-    binary: ocb
+    ignore:
+      - goos: windows
+        goarch: arm64
+    binary: builder
```

Comment:
I think this change deserves its own issue and discussion. I would rather change the documentation, to be honest.

Reply:
@jpkrohling I am not sure I understand what is the comment about, and what is the resolution for that. Can you check if this PR can be merged.

Reply:
I would agree with @jpkrohling that `ocb` is a better name. `builder` sounds vague to me, something I wouldn't add in $PATH as is

Reply:
dropped